pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -116,7 +117,13 @@ public void writeResource(HapiRowDescriptor element)
       meta.addTag(
           new Coding(removeAction.getSystem(), removeAction.toCode(), removeAction.getDisplay()));
     } else {
-      resource = (Resource) parser.parseResource(jsonResource);
+      try {
+        resource = (Resource) parser.parseResource(jsonResource);
+      }
+      catch (DataFormatException e) {
+        e.printStackTrace();
```

Comment:
Please use `log.error` instead. Also if you can count the number of these in a new counter that would be great (see the above `METRICS_NAMESPACE` counters).

Reply:
Hi Bashir ,
I have made the necessary changes.